### ISSUE TITLE
doesn't work with dietary delights

### ISSUE THREAD

**Issue by TheBearodactyl** [0vergrown/Fairy-Origin#3]
i installed dietary delights and i still can't eat things like cabbage as a fairy

**Comment by 0vergrown** [comment]
> i installed dietary delights and i still can't eat things like cabbage as a fairy

What mod is the cabbage from?

**Comment by TheBearodactyl** [comment]
> > i installed dietary delights and i still can't eat things like cabbage as a fairy
> 
> What mod is the cabbage from?

farmers delight

**Comment by TheBearodactyl** [comment]
I CAN'T EAT CARROTS APPARENTLY

**Comment by 0vergrown** [comment]
> I CAN'T EAT CARROTS APPARENTLY

Yeah, that's normal. All of that is intended from the Fairy as listed by the following power:

"Nature's Diet: Bound by their ethereal essence, a Fae's palate is attuned solely to the pure nourishment of nature. While the world offers a plethora of foods, they find sustenance only in the sweet bounty of fruits and berries. Any attempt to consume other foods proves futile, as their delicate constitution rejects anything that doesn't resonate with the natural world."

The fairy is also kinda isolated in their diet, Dietary Delights was made years after the Fairy origin, so compatibility with it wasn't in mind. I most likely update the Fairy to make its food tags more global and maybe update Dietary Delights to have some type of tag that fits that type of Origin, but we'll see on that, don't even know what to call that type of tag 😂
